cpanfile

```diff
@@ -11,7 +11,7 @@ requires 'Scalar::Util';
 requires 'Unicode::Normalize', '>= 1.25';
 requires 'DataDog::DogStatsd::Helper', '>= 0.05';
 requires 'Syntax::Keyword::Try', '>= 0.04';
-requires 'perl', '5.014';
+requires 'perl', '5.024';
```

Comment:
Changing the minimum Perl version should also be mentioned in the changelog please